openstates/ne/legislators.py

```diff
@@ -30,12 +30,18 @@ def scrape(self, term, chambers):
                     phone = phone[1] + ' ' + phone[2]
                 else:
                     phone = None
-                email = page.xpath('//div[@id="sidebar"]/ul[1]/li[7]/a')[0].text or ''
+                mailto = page.xpath('//div[@id="sidebar"]/ul[1]/li[contains(text(), "Email:")]/a/@href')[0]
+                email = mailto[7:]
+
+                photo_url = \
+                        "http://www.nebraskalegislature.gov/media/images/blogs/dist%d02.jpg" \
```

Comment:
These images don't exist. It looks like it works if you remove the "02" after %d.